src/services/CertificateGenerationService.ts

```diff
@@ -589,7 +589,7 @@ class CertificateGenerationService {
           bodyType: testResult.bodyType?.description,
           date: moment(testResult.testTypes.testTypeStartTimestamp).format("DD/MM/YYYY"),
           testerName: testResult.testerName,
-          reapplicationDate: this.calculateVehicleApprovalRetestDate(testResult.testTypes.testTypeStartTimestamp),
+          reapplicationDate: testResult.reapplicationDate ?? '',
```

Comment:
this doesn't cover all the ac's
"need to handle that the field may not exist, may not contain a a value, or may be null or undefined"